data/upgrades/astromech.json

```diff
@@ -9,21 +9,14 @@
         "type": "Astromech",
         "ability": "Action: Spend 1 non-recurring [Charge] from another equipped upgrade to recover 1 shield. Action: Spend 2 shields to recover 1 non-recurring [Charge] on an equipped upgrade.",
         "images": "https://images-cdn.fantasyflightgames.com/filer_public/5d/1a/5d1a60e4-50f6-40fc-9711-4c4f4513eea9/swz06-chopper-astromech.png",
-        "slots": [
-          "Astromech"
-        ]
+        "slots": ["Astromech"],
+        "image": "https://sb-cdn.fantasyflightgames.com/card_images/Card_Upgrade_99.png",
+        "artwork": "https://sb-cdn.fantasyflightgames.com/card_art/Card_art_XW_U_99.jpg"
```

Comment:
The artwork is from the Chopper crew card, not the astromech. I see this is a problem in the FFG app :(

Can you remove the `artwork` field for now until FFG fixes their mistake?

Reply:
Actually, I'll just merge this PR and do this change myself. No need to block this PR for it :)